### PR TITLE
Improve project.xml documentation.

### PR DESCRIPTION
**Pull request by player-03**
Including information about new features coming in Lime 8.1.0.

### CODE REVIEW

File: _docs/project-files/xml-format.md [hunk 1]
```diff
@@ -27,49 +27,49 @@ First, every node in the project file format supports `if` and `unless` attribut
  * ios, android, windows, mac, linux or html5
  * cpp, neko, flash or js
 
-You can `<set />` or `<unset />` values for conditional logic:
```
Comment (joshtynjala) [inline _docs/project-files/xml-format.md:30]: Even if it's not right here, I think the section for set should still mention unset.

Similarly, the define section should probably mention undefine.

Comment (joshtynjala) [reply _docs/project-files/xml-format.md:30]: Oops! I realized this is the general conditionals section, and not specifically for set. Perhaps just a casual mention of unset and undefine here is good enough.

Comment (player-03) [reply _docs/project-files/xml-format.md:30]: They actually weren't mentioned in the `<set />` and `<define />` sections, so I added that too. How's it look now?

Comment (joshtynjala) [reply _docs/project-files/xml-format.md:30]: Looks good!

**Pull request merged**